dubbo-metadata/dubbo-metadata-report-redis/src/test/java/org/apache/dubbo/metadata/store/redis/RedisMetadataReportTest.java

```diff
@@ -76,6 +87,10 @@ public void tearDown() throws Exception {
         this.redisServer.stop();
     }
 
+    private boolean isWindowsPlatform() {
```

Comment:
apache common lang has a tool named 'SystemUtils' to do this check. 
consider this is a unit test, I suggest add it as test scope dependency and use https://commons.apache.org/proper/commons-lang/apidocs/index.html?org/apache/commons/lang3/SystemUtils.html

Reply:
got it, i will fix it in the latest version